cloudbuild/pr-checks.yaml

```diff
@@ -24,14 +24,13 @@ steps:
       - |
         curl -sSL https://install.python-poetry.org | python3 - && \
         export PATH="/builder/home/.local/bin:$$PATH" && \
-        make setup && \
+        make install && \
         git init && \
         git add . && \
         make pre-commit && \
-        make compile-pipeline pipeline=training && \
-        make compile-pipeline pipeline=prediction && \
-        make setup-all-components && \
-        make test-all-components
+        make compile pipeline=training && \
+        make compile pipeline=prediction && \
```

Comment:
@browningjp-datatonic shall we build containers here, too?

Reply:
I thought about this and decided not to - it will slow down the builds / cost more in compute for not a lot of benefit. The container images will be built for the E2E test pipeline, so it will get checked on PR anyway